oauthproxy_test.go

```diff
@@ -1473,11 +1519,13 @@ func TestAjaxForbiddendRequest(t *testing.T) {
 }
 
 func TestClearSplitCookie(t *testing.T) {
-	opts := options.NewOptions()
+	opts := baseTestOptions()
 	opts.Cookie.Name = "oauth2"
 	opts.Cookie.Domains = []string{"abc"}
-	store, err := cookie.NewCookieSessionStore(&opts.Session, &opts.Cookie)
-	assert.Equal(t, err, nil)
+	cipher, err := encryption.NewBase64Cipher(encryption.NewCFBCipher, encryption.SecretBytes(opts.Cookie.Secret))
```

Comment:
I think it would be advantageous in 1 of these 2 tests to flip to the base64 version of the cookie secret option to get coverage of both formats

Reply:
I've set this one to use the base64. I know what you mean, but ideally it shouldn't matter as we are testing that elsewhere, long term I'd like to see an end to end suite that allows us to flex this kind of thing, long term meaning very long term 😅